src/components/input_fields/code_field.tsx

```diff
@@ -189,22 +196,24 @@ export default class CodeField extends React.Component<CodeFieldProps, CodeField
      * Gets a function that will handle special keyboard keys.
      * @param index - The index of the field in which the key was pressed.
      * @param key - The name of the keyboard key that was pressed.
-     * @return - Null if the key was special; otherwise a function that will return true if the default behavior should
-     * be blocked;
+     * @return - Null if the key was special; otherwise a function that will return true if the
+     * default behavior should be blocked;
      */
     private _getSpecialKeyHandler(index: number, key: string): () => boolean | null {
         const { type, maxLength } = this.props;
         const { values } = this.state;
-        /** a mapping of keys that should be handled. Each function returns whether the event was handled. **/
+
         return {
             // arrow left & right let you move between the fields
             ArrowLeft: () => this._focusOnPromptInput(index - 1),
             ArrowRight: () => this._focusOnPromptInput(index + 1),
             // arrow up / down let you increase / decrease numbers
-            ArrowUp: () => type == 'number' &&
-                this._changeValue(index, values[index] && ((parseInt(values[index]) + 1) % 10).toString(), -1),
-            ArrowDown: () => type == 'number' &&
-                this._changeValue(index, values[index] && ((parseInt(values[index]) + 9) % 10).toString(), -1),
+            ArrowUp: () => type === 'number'
+                && this._changeValue(index, values[index]
+                    && ((parseInt(values[index], 10) + 1) % 10).toString(), -1),
```

Comment:
why this weird alignment?

Reply:
they should be under each other, it most have got confused.